pkg/virt-config/feature-gates.go

```diff
@@ -109,161 +40,161 @@ func (config *ClusterConfig) isFeatureGateEnabled(featureGate string) bool {
 }
 
 func (config *ClusterConfig) WorkloadEncryptionSEVEnabled() bool {
-	return config.isFeatureGateEnabled(WorkloadEncryptionSEV)
+	return config.isFeatureGateEnabled(featuregate.WorkloadEncryptionSEV)
 }
 
 func (config *ClusterConfig) DockerSELinuxMCSWorkaroundEnabled() bool {
-	return config.isFeatureGateEnabled(DockerSELinuxMCSWorkaround)
+	return config.isFeatureGateEnabled(featuregate.DockerSELinuxMCSWorkaround)
 }
 
 func (config *ClusterConfig) VSOCKEnabled() bool {
-	return config.isFeatureGateEnabled(VSOCKGate)
+	return config.isFeatureGateEnabled(featuregate.VSOCKGate)
 }
 
 func (config *ClusterConfig) CustomSELinuxPolicyDisabled() bool {
-	return config.isFeatureGateEnabled(DisableCustomSELinuxPolicy)
+	return config.isFeatureGateEnabled(featuregate.DisableCustomSELinuxPolicy)
 }
 
 func (config *ClusterConfig) MediatedDevicesHandlingDisabled() bool {
-	return config.isFeatureGateEnabled(DisableMediatedDevicesHandling)
+	return config.isFeatureGateEnabled(featuregate.DisableMediatedDevicesHandling)
 }
 
 func (config *ClusterConfig) KubevirtSeccompProfileEnabled() bool {
-	return config.isFeatureGateEnabled(KubevirtSeccompProfile)
+	return config.isFeatureGateEnabled(featuregate.KubevirtSeccompProfile)
 }
 
 func (config *ClusterConfig) HotplugNetworkInterfacesEnabled() bool {
-	return config.isFeatureGateEnabled(HotplugNetworkIfacesGate)
+	return config.isFeatureGateEnabled(featuregate.HotplugNetworkIfacesGate)
 }
 
 func (config *ClusterConfig) PersistentReservationEnabled() bool {
-	return config.isFeatureGateEnabled(PersistentReservation)
+	return config.isFeatureGateEnabled(featuregate.PersistentReservation)
 }
 
 func (config *ClusterConfig) VMPersistentStateEnabled() bool {
-	return config.isFeatureGateEnabled(VMPersistentState)
+	return config.isFeatureGateEnabled(featuregate.VMPersistentState)
 }
 
 func (config *ClusterConfig) MultiArchitectureEnabled() bool {
-	return config.isFeatureGateEnabled(MultiArchitecture)
+	return config.isFeatureGateEnabled(featuregate.MultiArchitecture)
 }
 
 func (config *ClusterConfig) VMLiveUpdateFeaturesEnabled() bool {
-	return config.isFeatureGateEnabled(VMLiveUpdateFeaturesGate)
+	return config.isFeatureGateEnabled(featuregate.VMLiveUpdateFeaturesGate)
 }
 
 func (config *ClusterConfig) BochsDisplayForEFIGuestsEnabled() bool {
-	return config.isFeatureGateEnabled(BochsDisplayForEFIGuests)
+	return config.isFeatureGateEnabled(featuregate.BochsDisplayForEFIGuests)
 }
 
 func (config *ClusterConfig) NetworkBindingPlugingsEnabled() bool {
-	return config.isFeatureGateEnabled(NetworkBindingPlugingsGate)
+	return config.isFeatureGateEnabled(featuregate.NetworkBindingPlugingsGate)
 }
 
 func (config *ClusterConfig) AutoResourceLimitsEnabled() bool {
-	return config.isFeatureGateEnabled(AutoResourceLimitsGate)
+	return config.isFeatureGateEnabled(featuregate.AutoResourceLimitsGate)
 }
 
 func (config *ClusterConfig) CommonInstancetypesDeploymentEnabled() bool {
```

Comment:
FWIW I noticed this wasn't actually being called anywhere in https://github.com/kubevirt/kubevirt/pull/12753 and wanted to follow up by enablding linting. Given you're refactoring so much here would you mind enabling it for this package now? Happy to handle it myself in a follow up if not.